test/index.js

```diff
@@ -0,0 +1,38 @@
+var lastChar = require('../lib')
+  , assert = require('assert')
+  ;
+
+describe('last-char', function () {
```

Comment:
That would be amazing! :sparkle: Or maybe is it possible to generate the code using some tool?

I'm not documented in this direction, but I'm interested to find solutions.